security/protect-hosts.md

```diff
@@ -20,6 +20,7 @@ This how-to guide uses the following {{site.prodname}} features:
 - **FelixConfiguration** resource with parameters:
   - **FailsafeInboundHostPorts**
   - **DefaultEndpointToHostAction**
+- **KubeControllersConfiguration**
 
 ### Concepts
```

Comment:
Consider adding a line or two in Concepts section, "Host endpoints" about how you can also "configure manage lifecycle of host endpoints" and why you would want to do that. That is, if I don't configure the kubecontroller config what is the downside? Something like, "if you do not configure kubecontroller to handle manage the hostendpoint upon creation, you must rely on your own manual process to protect your Kubernetes nodes upon creation."